proposal/veax-void/README.md

```diff
@@ -0,0 +1,46 @@
+# :pig: PIGS :pig:
+---
+
+Readme for PIGS or Pixelwise Image Graph Segmentation project developmet.
+
+## Basic information
+[Official PIGS repo](https://github.com/veax-void/nsdhw_21sp/tree/veax-void-proposal)
+
+Fast package for image segmentatioin using graphs, like your dad use to make! 
+
+## Problem to solve
+
+Image processing, especially segmentation tasks are really heavy on computations, but also provide a lot of value. 
+Graph partitioning provides a powerful approach to image segmentation.  
+I propose to use fast cpp code for make computation of relations in the graph more efficient.  
+
+**TODO**: add math and algorithm
+
+## Perspective users
+All people who use image data representations mostly. But this package for programmers who want to integrate fast graph segmentation in to their projects.
+
+## System architecture
+
+**TODO**: Make a system dataflow, add charts!
+
+## API
+**TODO**: Add API
+
+## Infrastructure
+**TODO**: TESTS, DOCS, version control, CI/CD
+
+
+## Schedule
+today: 28/03 Days to go: 70  
+- 5 days: Research
+- 30 days: Development
```

Comment:
A good starting point.  You may add some sub-bullets on what you plan to develop.  Also, put dates on each of the milestones.